charmcraft/commands/store/__init__.py

```diff
@@ -861,7 +881,7 @@ def run(self, parsed_args):
 
         for lib_data in to_publish:
             store.create_library_revision(
-                lib_data.charm_name, lib_data.lib_id, lib_data.api, lib_data.patch,
+                charm_name, lib_data.lib_id, lib_data.api, lib_data.patch,
```

Comment:
But `lib_data.charm_name` should be equal to `charm_name`, right? this is even validated above..

Reply:
Ah yes, this is a mistake from an earlier iteration. Removed.